reblog/env.py

```diff
@@ -0,0 +1,12 @@
+"""
+This file contains convenience methods and hardcoded paths depending on usage
+"""
+__author__ = "Varun Nayyar <nayyarv@gmail.com>"
+
+import os
+
+if "CI" in os.environ:
+    FULL = True
+else:
+    print("FULL mode off")
+    FULL = False
```

Comment:
Hmm... I really don't know much about what this is doing here, but looks important.

Reply:
This isn't perfect, but it was meant to allow us to do partial versions while developing while the full version would be built on the CI server. Github inserts environment variables that can accessed and this was meant to be a way of switching between one size or another in an easy way. 

Basically my problem was that I'd make a minor change, do a jupyter build and then have to wait 15 minutes. This allows notebooks to have a dev setting which is a shorter than the full build which allows one to test the build and aesthetics without needing more stuff.